WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -122,6 +122,37 @@
         });
       </script>
 
+
+      <% if @competition_admin_view %>
```

Comment:
I'd definitely like to show this to everyone.

Corner cases:

- What happens if this competition does not yet have a location?
- What happens if this competition does not yet have a date scheduled?
- What happens if there are no nearby competitions? Does it look weird?